app/swagger/requests/in_progress_forms.rb

```diff
@@ -55,13 +55,7 @@ class InProgressForms
         operation :get do
           extend Swagger::Responses::AuthenticationError
 
-          key :summary,     'Get form data'
```

Comment:
do you mean for this swagger to no longer have a `summary`?

Reply:
yep, since the summary and description are now the same I thought I'd return this to just having a description